src/sider/manglendemedvirkning/unntak/UnntakSkjema.tsx

```diff
@@ -29,6 +29,7 @@ const texts = {
     "Åpne forhåndsvisning for å se vurderingen. Når du trykker Lagre journalføres vurderingen automatisk.",
   forhandsvisningLabel: "Forhåndsvis vurderingen",
   missingBegrunnelse: "Vennligst angi begrunnelse",
+  bruker: "Husk å informere bruker om utfallet av vurderingen.",
```

Comment:
Synes det er nyttig om variabelen i `texts` kan fortelle litt om hva som står i den slik at man allerede på denne linjen `<BodyShort>{texts.bruker}</BodyShort>` får en indikasjon på hva teksten handler om 😊
Hva med noe som ligner på dette? 👇😊
```suggestion
  informerBruker: "Husk å informere bruker om utfallet av vurderingen.",
```
eventuelt
```
  informUser: "Husk å informere bruker om utfallet av vurderingen.",
```